Commands/Public/ping.js

```diff
@@ -0,0 +1,24 @@
+const { Stopwatch } = require("../../Modules/Utils/");
+
+module.exports = async ({ client, Constants: { Colors } }, documents, msg, commandData) => {
+	const timer = new Stopwatch();
+	await msg.send({
+		embed: {
+			color: Colors.INFO,
+			title: "Getting the ping 🏓",
+			description: "Please stand by...",
+		},
+	});
+	const sendPing = timer.duration;
+	timer.stop();
+	msg.send({
+		embed: {
+			color: Colors.LIGHT_GREEN,
+			title: "Pong! 🏓",
```

Comment:
I mean if we want to be consistent, we can make this: "Received the pong"

Reply:
You must be fun at parties.